cmd/checkmetrics/ci_slaves/checkmetrics-json-clh-baremetal.toml

```diff
@@ -0,0 +1,47 @@
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file contains baseline expectations
+# for checked results by checkmetrics tool.
+#
+# values set specifically for packet.com c1.small slave.
+
+[[metric]]
+name = "boot-times"
+type = "json"
+description = "measure container lifecycle timings"
+# Min and Max values to set a 'range' that
+# the median of the CSV Results data must fall
+# within (inclusive)
+checkvar = ".\"boot-times\".Results | .[] | .\"to-workload\".Result"
+checktype = "mean"
+midval = 1.60
+minpercent = 5.0
+maxpercent = 5.0
+
+[[metric]]
+name = "memory-footprint"
+type = "json"
+description = "measure container average footprint"
+# Min and Max values to set a 'range' that
+# the median of the CSV Results data must fall
+# within (inclusive)
+checkvar = ".\"memory-footprint\".Results | .[] | .average.Result"
+checktype = "mean"
+midval = 283923.70
```

Comment:
I agree with @devimc, these numbers look particularly high. But the explanation might be the usage of `virtio-fs`. What are the numbers if you enable `virtio-fs w/ DAX` for QEMU?